app/models/personal_information_log.rb

```diff
@@ -2,4 +2,10 @@
 
 class PersonalInformationLog < ActiveRecord::Base
   validates(:data, :error_class, presence: true)
+
+  def decoded_data
```

Comment:
Generally minor nit around separation of concerns, but `decoded_data` won't be useful for the full scope (HCA submission logging, at present). Should we subclass for the request/response logging and add this functionality there, or break it out into a routine specific to an analysis rake task of some sort?

Reply:
yeah, I realized this method is only useful for our general logging case. It's seamless if not applicable (just reflects `data`)... I'd lean towards a TODO to remove when we figure out analysis approach as opposed to subclassing. I wanted something short term to work with data in rails console if needed